webapp/src/view/favourites/favourite.tsx

```
@@ -1,38 +1,52 @@
-                    </Grid>
-                ))
-            ) : (
-                <Grid item xs={12}>
-                    <ErrorHandler message="No favourite applications were found" />
-                </Grid>
-            )}
+  return (
+    <Grid container spacing={2} sx={{ width: "100%" }}>
+      {favApps.length > 0 ? (
+        favApps.map((app) => (
+          <Grid size={{ xs: 12, sm: 6, md: 4, lg: 2.6 }}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)